templates/Includes/MainNav.ss

```diff
@@ -21,7 +21,7 @@
                                 <ul class="dropdown-menu" role="menu" aria-hidden="true">
                                     <% loop $Children %>
                                         <li role="menuitem" <% if $LinkingMode = current %>class="active"<% end_if %>>
-                                            <a role="menuitem" class="dropdown-item" href="$Link" <% if $LinkingMode = current %>aria-label="current page" <% end_if %>>
+                                            <a role="menuitem" class="dropdown-item" href="$Link" <% if $LinkingMode = current %>aria-current="page" <% end_if %>>
```

Comment:
Does it make sense for the value of `aria-current` to still be `page`?

Reply:
What would you suggest otherwise sorry?

Reply:
Looks like it makes sense. I didn't really know anything about `aria-current` but I've looked it up now.

Reply:
Sorry I should of linked to https://github.com/silverstripe/cwp-watea-theme/pull/76 for more info